tests/app/db.py

```diff
@@ -836,28 +836,28 @@ def ses_complaint_callback():
 def ses_notification_callback():
     return (
         '{\n  "Type" : "Notification",\n  "MessageId" : "ref1",'
-        '\n  "TopicArn" : "arn:aws:sns:eu-west-1:123456789012:testing",'
+        '\n  "TopicArn" : "arn:aws:sns:eu-west-2:123456789012:testing",'
         '\n  "Message" : "{\\"notificationType\\":\\"Delivery\\",'
         '\\"mail\\":{\\"timestamp\\":\\"2016-03-14T12:35:25.909Z\\",'
         '\\"source\\":\\"test@test-domain.com\\",'
-        '\\"sourceArn\\":\\"arn:aws:ses:eu-west-1:123456789012:identity/testing-notify\\",'
+        '\\"sourceArn\\":\\"arn:aws:ses:eu-west-2:123456789012:identity/testing-notify\\",'
         '\\"sendingAccountId\\":\\"123456789012\\",'
         '\\"messageId\\":\\"ref1\\",'
         '\\"destination\\":[\\"testing@digital.cabinet-office.gov.uk\\"]},'
         '\\"delivery\\":{\\"timestamp\\":\\"2016-03-14T12:35:26.567Z\\",'
         '\\"processingTimeMillis\\":658,'
         '\\"recipients\\":[\\"testing@digital.cabinet-office.gov.uk\\"],'
         '\\"smtpResponse\\":\\"250 2.0.0 OK 1457958926 uo5si26480932wjc.221 - gsmtp\\",'
-        '\\"reportingMTA\\":\\"a6-238.smtp-out.eu-west-1.amazonses.com\\"}}",'
+        '\\"reportingMTA\\":\\"a6-238.smtp-out.eu-west-2.amazonses.com\\"}}",'
         '\n  "Timestamp" : "2016-03-14T12:35:26.665Z",\n  "SignatureVersion" : "1",'
         '\n  "Signature" : "X8d7eTAOZ6wlnrdVVPYanrAlsX0SMPfOzhoTEBnQqYkrNWTqQY91C0f3bxtPdUhUt'
         "OowyPAOkTQ4KnZuzphfhVb2p1MyVYMxNKcBFB05/qaCX99+92fjw4x9LeUOwyGwMv5F0Vkfi5qZCcEw69uVrhYL"
         "VSTFTrzi/yCtru+yFULMQ6UhbY09GwiP6hjxZMVr8aROQy5lLHglqQzOuSZ4KeD85JjifHdKzlx8jjQ+uj+FLzHXPMA"
         'PmPU1JK9kpoHZ1oPshAFgPDpphJe+HwcJ8ezmk+3AEUr3wWli3xF+49y8Z2anASSVp6YI2YP95UT8Rlh3qT3T+V9V8rbSVislxA==",'
-        '\n  "SigningCertURL" : "https://sns.eu-west-1.amazonaws.com/SimpleNotificationService-bb750'
+        '\n  "SigningCertURL" : "https://sns.eu-west-2.amazonaws.com/SimpleNotificationService-bb750'
```

Comment:
The certificate for eu-west-2 is almost certainly not the same as https://sns.eu-west-1.amazonaws.com/SimpleNotificationService-bb750dd426d95ee9390147a5624348ee.pem

Reply:
Similar to the SES endpoints/ARNs, the SNS related issues will be handled in an upcoming ticket.